.gitignore

```diff
@@ -225,3 +226,6 @@ flycheck_*.el
 
 !/.github/
 !/.github/**
+
+# vscode
+.vscode/
```

Comment:
Wow, surprising that the company-wide .gitignore file doesn't have this already... Can you put this above the line that says "BEGIN managed content"?